heroku/cli
# feat(cli): Adding disclaimer for plugin AI install

## Description

Here we're adding a disclaimer notice to be shown when the AI plugin gets installed.

## Test instructions
- Download this branch and build.
- Run `./bin/run plugins:install events`, no disclaimer should be displayed.
- Run `./bin/run plugins:install @heroku/plugin-ai`, you should see the disclaimer and a prompt. If you accept the prompt, installation should continue, if you reject the prompt an error should be raised and installation canceled.

## SOC2 Compliance
GUS Work Item: [W-17041683](https://gus.lightning.force.com/a07EE000023rqE1YAI)
